tasks/clientlibs.js

```diff
@@ -377,17 +387,95 @@ module.exports = function (grunt) {
 		 * @returns {object} The sorted array
 		 */
 		function performSort(arr) {
-			var returnValue = null,
-				arrLength = arr.length,
-				i = 0;
+			var arrLength = arr.length,
+				i = 0,
+				resultArray = [],
+				item = {};				
+				
+			for (i = 0; i < arrLength; i++) {
```

Comment:
Inconsistency on how we are doing i = i + 1 vs i++ (or j = j + 1 or j++)

Reply:
I don't think there is a standard saying which one is better, just for personal preference, respect Dale is the author for this plugin, since he prefers for i = i + 1, I will update my code.